pom.xml

```diff
@@ -94,14 +95,28 @@
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_${scala.binary.version}</artifactId>
             <version>3.1.0</version>
-            <scope>test</scope>
+            <optional>true</optional>
         </dependency>
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
             <version>2.6</version>
             <optional>true</optional>
         </dependency>
+
+        <!-- Test scope dependencies -->
+
+        <dependency>
+            <groupId>org.scalatestplus</groupId>
+            <artifactId>mockito-1-10_${scala.binary.version}</artifactId>
+            <version>3.1.0.0</version>
+        </dependency>
```

Comment:
This dependency is commented to be in the `test` scope, yet the dependency itself is in the default (`compile`) scope.

Reply:
good catch ! :)

Reply:
Sometime comments in code aren't that useless :smile: